courses/serializers.py

```diff
@@ -152,6 +159,62 @@ def get_topics(self, instance):
             key=lambda topic: topic["name"],
         )
 
+    def get_page(self, instance):
+        return (
+            CoursePageSerializer(
+                instance=CoursePage.objects.filter(course=instance).get()
+            ).data
+            if CoursePage.objects.filter(course=instance).exists()
+            else None
+        )
+
+    def get_page_url(self, instance):
+        page = self.get_page(instance)
+
+        return None if page is None else page["page_url"]
+
+    def get_description(self, instance):
+        page = CoursePage.objects.filter(course=instance).first()
+        return bleach.clean(page.description, tags=[], strip=True) if page else None
+
+    def get_current_price(self, instance):
+        next_run = self.get_next_run_id(instance)
+
+        if next_run is None:
+            return None
+
+        course_ct = ContentType.objects.get(app_label="courses", model="courserun")
+
+        relevant_product = (
+            Product.objects.filter(
+                content_type=course_ct, object_id=next_run, is_active=True
+            )
+            .order_by("-price")
+            .first()
+        )
+        return relevant_product.price if relevant_product else None
+
+    def get_instructors(self, instance):
+        page = CoursePage.objects.filter(course=instance).first()
+
+        if page is None:
+            return None
+
+        members = [member.value for member in page.faculty_members]
+        returnable_members = []
+
+        for member in members:
+            returnable_members.append(
+                {
+                    "name": member["name"],
+                    "description": bleach.clean(
+                        member["description"].source, tags=[], strip=True
+                    ),
+                }
+            )
+
+        return returnable_members
```

Comment:
This can be moved to `CoursePageSerializer` as well. But I am unsure if we should include this in all API responses for the course page. Maybe we can add a flag like `include_page_fields ` in `CoursePageSerializer`.  WDYT?